m3u8/parser.py

```diff
@@ -111,10 +111,13 @@ def parse(content, strict=False, custom_tags_parser=None):
             state['cue_out'] = True
 
         elif line.startswith(protocol.ext_x_cue_out):
-            _parse_cueout(line, state, string_to_lines(content)[lineno - 2])
+            _parse_cueout(line, state)
```

Comment:
This change allowed me to simplify "cue out" parsing. We no longer require the "previous line" to be examined.

Reply:
You are right! Much better.